test/extended/oauth/sampleoauthserver.go

```diff
@@ -36,29 +36,13 @@ var _ = g.Describe("[Suite:openshift/oauth/run-oauth-server] Run the integrated
 		oc = exutil.NewCLI("oauth-server-configure", exutil.KubeConfigPath())
 	)
 
-	// g.It("should successfully be configured and be responsive", func() {
-	// 	serverAddress, cleanup, err := exutil.DeployOAuthServer(oc, []osinv1.IdentityProvider{}, nil, nil)
-	// 	defer cleanup()
-	// 	o.Expect(err).ToNot(o.HaveOccurred())
-	// 	e2e.Logf("got the OAuth server address: %s", serverAddress)
-
-	// 	http.DefaultTransport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: true} // FIXME: VERY VERY UGLY, don't do this at home
-	// 	resp, err := http.Get(serverAddress)
-	// 	o.Expect(err).ToNot(o.HaveOccurred())
-	// 	defer resp.Body.Close()
-
-	// 	body, err := ioutil.ReadAll(resp.Body)
-	// 	e2e.Logf("The body received: %s", string(body))
-	// 	o.Expect(err).ToNot(o.HaveOccurred())
-	// })
-
-	g.It("should successfully be configured and be responsive", func() {
+	g.It("should successfully configure htpasswd and be responsive", func() {
 		secrets := []corev1.Secret{{
 			ObjectMeta: metav1.ObjectMeta{
 				Name: "htpasswd-secret",
 			},
 			Data: map[string][]byte{
-				"htpasswd": []byte("testuser:$2y$05$pOYBCbJ1RXr.vDzPXdyTxuE96Nojc9dNI9R3QjkWUj2t/Ae/jmFy."), // userinfo testuser:password
+				"htpasswd": []byte("testuser:$apr1$iD9QmkLW$dfVpx1X7533hKAVSiRfhd1"), // userinfo testuser:password
```

Comment:
The above was correct I believe, use -B for bcrypt hashes while generating the htpasswd file (interesting fact - nginx does not support bcrypt hashes for htpasswd files for its basic auth)

Reply:
yea, i figured you knew what you were doing : )